lib/web/apiserver.go

```diff
@@ -1258,6 +1260,10 @@ type CreateSessionResponse struct {
 	TokenExpiresIn int `json:"expires_in"`
 	// SessionExpires is when this session expires.
 	SessionExpires time.Time `json:"sessionExpires,omitempty"`
+	// WebIdleTimeout specifies how long a user WebUI session can be left
+	// idle before being logged out by the server. A zero value means there
+	// is no idle timeout set.
+	WebIdleTimeout int `json:"webIdleTimeout"`
```

Comment:
`WebIdleTimeout` -> `SessionInactivityTimeout` json `sessionInactivityTimeout` 

No need to add an extra `Web` prefix as each field here is related to web, makes it strange from the client side perspective.